CliClient/locales/es_ES.po

```diff
@@ -737,14 +737,14 @@ msgstr "La versión actual está actualizada."
 
 #, javascript-format
 msgid "%s (pre-release)"
-msgstr ""
+msgstr "%s (pre-lanzamiento)""
```

Comment:
Hi @tanrax, there was an error here (an extra quote), which made the build fail. I wonder how you've edited po file? Was it manually or did you use some tool and, if so, which one? In general it's strongly recommended not to manually edit this file, but also if you used a tool it would be good to know which one, so that I can recommend against it in the readme. Thanks!

Reply:
I created a new pull resquest by fixing it. https://github.com/laurent22/joplin/pull/1257